modules/vnet_peering/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "local_resource_group_name" {
+  description = "Name of the existing local peer resource group where to place the resources created."
+  type        = string
+}
+
+variable "peer_resource_group_name" {
+  description = "Name of the existing remote peer resource group where to place the resources created."
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "Prefix name appended to the peering names."
+  default     = ""
+  type        = string
+}
+
+variable "local_vnet_name" {
+  description = "Local VNET name."
+  default     = ""
```

Comment:
I would say: no defaults here. The VNET name is mandatory, otherwise the module does not make sense.

```suggestion
```